src/ResourceManager/Resources/Commands.Resources/RoleAssignments/RemoveAzureRoleAssignmentCommand.cs

```diff
@@ -153,7 +153,7 @@ public class RemoveAzureRoleAssignmentCommand : ResourcesBaseCmdlet
         public SwitchParameter PassThru { get; set; }
 
         [ValidateNotNullOrEmpty]
-        [Parameter(Position = 0, Mandatory = true, ValueFromPipeline = true, ParameterSetName = ParameterSet.RoleAssignment, HelpMessage = "Role Assignment.")]
+        [Parameter(Position = 0, Mandatory = true, ValueFromPipeline = true, ParameterSetName = "RoleAssignment", HelpMessage = "Role Assignment.")]
```

Comment:
is this change necessary?

Reply:
I have to make this change, because  it's breaking my build.

Reply:
Please revert all ofthese changes outside the Compute project